audit-log/README.md

```diff
@@ -0,0 +1,151 @@
+--- 
+layout: pattern
+title: Audit Log
+folder: audit-log
+permalink: /patterns/audit-log/ 
+categories:
+- creational 
+  language: en 
+  tags:
+- Security
+- Data Access
+---
+
+## Name / classification
+
+Audit Log.
+
+## Also known as
+
+Audit trail.
+
+## Intent
+
+Have a simple log of changes, intended to be easily written and non-intrusive.
+
+## Explanation
+
+Real-world example
+
+> A customer changes their address, we want an independent record that this change occured, and 
+> when it has occurred.
+
+In plain words
+
+> Whenever a some property changes, log the change in a file.
+
+Wikipedia says
+
+> An audit trail (also called audit log) is a security-relevant chronological record, set of 
+> records, and/or destination and source of records that provide documentary evidence of the 
+> sequence of activities that have affected at any time a specific operation, procedure, event, 
+> or device.  
+
+**Programmatic Example**
```

Comment:
This does not need to be bolded. Also, make sure the paragraph starts on a new line

Reply:
Thank you for catching the paragraph issue. The bolded text is very common within the repository, so I will be leaving it as is.